db/tables.py

```diff
@@ -52,6 +54,19 @@ def create_mathesar_table(name, schema, columns_, engine, metadata=None):
     return table
 
 
+def delete_table(name, schema, engine):
```

Comment:
How does this handle references to this table from other tables? e.g. another table might have foreign keys pointing to this one.